llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -150,10 +172,12 @@ def __init__(
             del kwargs['name']
         if 'loss_fn' in kwargs:
             del kwargs['loss_fn']
-        if self.attn_config.get('alibi', False):
+        if self.attn_config.get('alibi', False) or self.attn_config.get(
```

Comment:
lets also check that alibi and rope are not both set?

Reply:
Alibi and rope are not mutually exclusive, someone can turn on both alibi and rope.

Reply:
Sorry, missed this comment. No, I don't think anybody has done this. I tried it [here](https://wandb.ai/mosaic-ml/rope_stability_lr_bs_125m/reports/loss-train-total-23-11-04-19-03-41---Vmlldzo1ODc0MDA5) and it gave slightly faster convergence initially, but converged to the same final training loss:

![image](https://github.com/mosaicml/llm-foundry/assets/144760128/ff3c20a4-ae08-4773-8992-fc1dc409f57c)